viewer/Viewer.cpp

```diff
@@ -140,6 +140,31 @@ namespace Enki
 		}
 	}
 	
+	void ViewerWidget::setCamera (QPointF pos, double altitude, double yaw, double pitch)
+	{
+		this->pos = pos;
```

Comment:
@plsm Can you please fix this?
@rmm-fcul I think this will affect the values we need to put into the Playground.cfg file.